contracts/SnappAuction.sol

```diff
@@ -8,6 +8,16 @@ contract SnappAuction is SnappBase {
     uint16 public constant AUCTION_BATCH_SIZE = 1000;
     uint8 public constant AUCTION_RESERVED_ACCOUNTS = 50;
     uint8 public constant AUCTION_RESERVED_ACCOUNT_BATCH_SIZE = 10;
+    
+    struct StandingOrders {
+        bytes32 orderHash;
+        uint validFrom;
+        uint validTo;
+    }
+
+    //mapping from accountId to nonce to StandingOrders
+    mapping (uint16 => mapping(uint128 => StandingOrders)) public reservedAccountOrders;
```

Comment:
the uint16 is not stored technically, it is just hashed.
I would take the solution, which is more readable, although yours is more gas efficient. But I am not sure, which one is more readable. 

Yeah, I will change to uint256. 

I used uint16 for reserved accounts, so that I do not have to do the transformation from uint16 to uint8 after calling this function:
>        uint16 accountId = publicKeyToAccountMap(msg.sender);

Storing as a unit8 would not make any gas difference, as this value gets hashed anyways and both values are below a wordsize

Reply:
> mapping(uint16 => (uint128, mapping(uint128 => StandingOrders))
checked the readablity. Does not look too shabby. Lets go with it.